pypesto/visualize/sampling.py

```diff
@@ -1,18 +1,44 @@
 import logging
 import matplotlib.pyplot as plt
 import matplotlib.axes
+from matplotlib.lines import Line2D
 import numpy as np
 import pandas as pd
 import seaborn as sns
-from typing import Sequence, Tuple
-
+from typing import Dict, Sequence, Tuple, Union
+
+from ..ensemble import (
+    get_percentile_label,
+    EnsemblePrediction,
+    MEDIAN,
+)
+from ..predict import PredictionResult
+from ..predict.constants import CONDITION, OUTPUT
 from ..result import Result
-from ..sample import McmcPtResult
+from ..sample import McmcPtResult, calculate_ci
+from .constants import (
+    LEN_RGB,
+    RGBA_BLACK,
+    RGBA_MIN,
+    RGBA_MAX,
+    RGB,
+)
+from .misc import rgba2rgb
+
 
 logger = logging.getLogger(__name__)
 
 
-def sampling_fval_trace(
+def sampling_fval_trace(*args, **kwargs):
+    logger.warning(
```

Comment:
maybe use warnings.warn for this, as this is sth that should be avoided in the first place https://stackoverflow.com/questions/9595009/warnings-warn-vs-logging-warning